src/service-worker.ts

```diff
@@ -0,0 +1,122 @@
+/// <reference lib="webworker" />
```

Comment:
Previous service worker content can be found here: http://rex-web.herokuapp.com/books/service-worker.js